src/App.js

```diff
@@ -313,7 +313,9 @@ class _AppStackWrapper extends React.Component {
     const version = DeviceInfo.getVersion();
     // We use this string to parse the version from user agent
     // in some of our services, so changing this might break another service
-    this.props.wallet.storage.config.setUserAgent(`Hathor Wallet Mobile / ${version}`);
+    if (this.props.wallet && this.props.wallet.storage) {
```

Comment:
:bulb:  This and the `updateReduxTokens` method can be simplified with an [optional chaining](https://javascript.info/optional-chaining#optional-chaining)
```suggestion
    if (this.props.wallet?.storage) {
```

Reply:
Done